src/views/Hero/index.view.tsx

```diff
@@ -16,11 +16,19 @@ const HeroView: React.FC<HeroProps> = ({ pageName, title, description }) => {
     <div
       className={cx("HeroView", { HeroView__homeView: pageName === "Home" })}
     >
-      <PostcardStack pageName={pageName} />
-      <EmailSubscriptionForm />
-      Hello from HeroView
-      {title}
-      {description}
+      <div className="HeroView__container">
+        <div className="HeroView__visual">
+          <PostcardStack pageName={pageName} />
+        </div>
+        <div className="HeroView__text">
+          <h1>CruzHacks 2021 /</h1>
+          <h1>
+            <span>{title}</span>
+          </h1>
+          <p>{description}</p>
+          <EmailSubscriptionForm />
+        </div>
+      </div>
```

Comment:
We should give the `h1` and `p` tags classnames. I am also not sure if we want to be nesting `p` tags within `h1` tags; rather, we can nest the `p` tag within a `div` instead. 

```suggestion
        <div className="HeroView__text">
          <h1 className="HeroView__titleText>CruzHacks 2021 /</h1>
          <h1 className="HeroView__titleTextTwo>{title}</h1>
          <p className="HeroView__description">{description}</p>
          <EmailSubscriptionForm />
        </div>
      </div>
```